src/back/responses.ts

```diff
@@ -428,14 +429,14 @@ export function registerRequestCallbacks(state: BackState): void {
   });
 
   state.socketServer.register(BackIn.DELETE_ALL_PLAYLISTS, async (event, req) => {
-    const playlists = await GameManager.findPlaylists();
+    const playlists = await GameManager.findPlaylists(state.preferences.browsePageShowExtreme);
```

Comment:
``DELETE_ALL_PLAYLISTS`` should delete all playlists even if extreme ones are hidden (since it's a developer tool at the DeveloperPage)